package messages;

option java_outer_classname = "_File_RobotLocation";
option java_multiple_files = true;



// robot locations should have a pose, and an uncertainty associated with it
message RobotLocation
{
    optional float x = 1;
    optional float y = 2;
    optional float h = 3;
    optional int64 timestamp = 4;
    optional float uncert = 5;
}
